netlify/edge-functions/dev-card.tsx

```diff
@@ -0,0 +1,381 @@
+import React from "react";
+import { ImageResponse } from "og_edge";
+import type { Config } from "https://edge.netlify.com";
+import { getLocalAsset } from "../og-image-utils.ts";
+
+const MAX_ABOUT_LENGTH = 77;
+const baseApiUrl = Deno.env.get("NEXT_PUBLIC_API_URL");
+
+function getTopPercent(oscr: number) {
```

Comment:
The functions are all in this file because we can import code from the Next.js codebase even though it's in the same repo. Netlify Edge functions occupy their own space.

Most of these functions have been copied or altered from the Next.js codebase.